README.md

```diff
@@ -1 +1,106 @@
-# c11-apollo-bandcamp-tracker
+# 🎸 Bandcamp Data Pipeline
+
+
+
+## 🔎 Overview
```

Comment:
maybe put a placeholder here for what this section will include, because I'm not sure how this is different from description at the moment

Reply:
ah ignore me, I've just seen that this is a header